paymentintent.go

```diff
@@ -44,6 +44,14 @@ const (
 	PaymentIntentStatusSucceeded            PaymentIntentStatus = "succeeded"
 )
 
+// PaymentIntentCaptureParams is the set of parameters that can be used when capturing a payment intent.
+type PaymentIntentCaptureParams struct {
```

Comment:
Is this a better approach than using the common one and adding the custom parameter `AmountToCapture` at the bottom like we do for Pay Invoice?

Reply:
I'm honestly not sure, but I have a slight preference for this way I think. Let's try it for now.